topics/migration_checklist.adoc

```diff
@@ -0,0 +1,28 @@
+=== Migration Checklist
+
+The following is a basic checklist of considerations to make before migrating content from an existing downstream repository to an upstream repository. In principle, the content should be laid out in accordance with one of the structures outlined in <<ccg-project-structure,Project Structure>>.
+
+Where will you submit the content?::
+Is there a dedicated upstream repository for documentation, or will the content be managed alongside the code? Alternatively, does a new upstream repository need to be created for documentation, and if so, who will perform this work?
+
+How will you submit the content?::
+When the content is ready, how should it be submitted to the upstream community? Can a standard pull request be submitted that adds all the content at the same time, or is there a different process?
+
+When will you submit the content?::
+Can the content be migrated at any time, or should it be migrated in line with an existing milestone such as an upstream community release? Who should be notified when the content has been migrated, and when will any new workflows commence? How will changes to the documentation be handled in the time after a pull request to submit the content has been created and before it has been accepted?
+
+Do you have the required permissions?::
+The members of the downstream documentation team must be granted the permissions and roles to work on the content in the upstream repository before any new workflows commence. Before requesting that any permissions or roles be granted, all members of the downstream documentation team must create an account on the appropriate website, if required.
+
+Have all brand-specific references been conditionalized?::
+If there are any references to resources that are inappropriate for or cannot be accessed by the upstream community, such as support statements or links to content on the Customer Portal, this content must be conditionalized so that it only appears in the downstream version of that content. Likewise, if there are any references to resources that are inappropriate for the end users of the downstream documentation, such as links to community documentation, this content must be conditionalized so that it only appears in the usptream version of that content.
```

Comment:
We might want to remove 'Customer Portal' and say 'internal support site' or something more generic.

Reply:
Typo: usptream

Reply:
Personally, I would prefer to keep the reference to 'Customer Portal'. I'm not sure that 'internal support site' will be readily understood by one of our major audiences - Red Hat associates - and given that this guidance is provided in the context of Red Hat rather than general community advice, I think it makes sense to refer to it. Perhaps we could clarify the term - call it the 'Red Hat Customer Portal', for example?

Reply:
Agreed. Clarity++.